packages/core/src/types.ts

```diff
@@ -1799,6 +1788,13 @@ export type ActorRefFrom<T> = ReturnTypeOrValue<T> extends infer R
     : never
   : never;
 
+export type ActorRefFromBehavior<T> = T extends Behavior<
+  infer TEvent,
+  infer TSnapshot
+>
+  ? ActorRef<TEvent, TSnapshot>
+  : never;
```

Comment:
This should be removed and the same capability should be added to `ActorRefFrom`